### PR TITLE
Add --dist-dir to every dist command when building for PyPI

### LINKED ISSUE #254: Tarballs are not uploaded to PyPI
It seems that rever is not uploading .tar.gz source distributions to PyPI when configured with

```
$PYPI_BUILD_COMMANDS = ['sdist', 'bdist_wheel']
```

I have this setting for two projects, and in both I need to manually `twine upload` the tarballs to PyPI.

Am I doing something wrong in these two rever scripts: [sage-flatsurf](https://github.com/flatsurf/sage-flatsurf/blob/master/rever.xsh) and [svgdigitizer](https://github.com/echemdb/svgdigitizer/blob/master/rever.xsh)?

For most users a wheel is perfectly fine. However, without the tarball, conda-forge does not pick up the new version automatically if its configured to download the tarball from PyPI.

### PR DESCRIPTION
**Pull request by saraedum**
fixes #254.

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 @pytest.fixture
 def gitrepo(request):
-    """A test fixutre that creates and destroys a git repo in a temporary
+    """A test fixture that creates and destroys a git repo in a temporary
```
Comment (saraedum) [inline tests/conftest.py:15]: I hope you don't mind that we add an unrelated typo fix that I stumbled upon.

**Comment by hadim** [comment]
@rjkoch @CJ-Wright you seem to be the latest persons who have pushed to this repo. Do you mind having a look? And potentially release a new version?

**Comment by saraedum** [comment]
@regro, sorry for the ping but could somebody have a look here please?

**Comment by saraedum** [comment]
@xhochy @ocefpaf @isuruf @CJ-Wright @marcelotrevisani , sorry for the ping but could somebody have a look here please?

**Comment by saraedum** [comment]
@regro , sorry for the ping but could somebody have a look here please?

**Comment by saraedum** [comment]
https://github.com/regro , sorry for the ping but could somebody have a look here please?

**Comment by ocefpaf** [comment]
> @xhochy @ocefpaf @isuruf @CJ-Wright @marcelotrevisani , sorry for the ping but could somebody have a look here please?

@saraedum I'm not a maintainer here. There is nothing I can do. Sorry.
